### DISCUSSION TITLE
Please add location where application was found

### DISCUSSION THREAD

**Discussion by jrg63** [Devolutions/UniGetUI#320] [General]
In the Info for an update, could you please add the path to where the application was found? Great program BTW, use it frequently

**Comment by marticliment** [comment]
This is not a thing that can easily be done, since package managers don't provide this kind of information.

I'm sorry :(

**Comment by jrg63** [comment]
Ah yes, I forgot your work is based on WinGet which doesn’t support this. Apologies

**Comment by Spacellary** [comment]
@martinet101 
One the reasons I still install some stuff using the Terminal is because I can read the log output and figure out where the things are going, is there a way to see the WingetUI log live _as it installs/uninstalls_, like a terminal window/box/panel/side-panel (like in ChocoGUI for example)?
The progress bar _alone_ isn't terribly useful, I'm afraid.

**Reply by marticliment** [reply]
You should be able to see the last line of the console on the textbox near to the progressbar, but currently there sno wya to see the whole live output of the installer. Could you please open a feature request?

**Reply by Spacellary** [reply]
👍
